rules/application/jvm/java_jndi_injection.yml

```diff
@@ -0,0 +1,24 @@
+title: Potential JNDI Injection Exploitation In JVM Based Application
+id: bb0e9cec-d4da-46f5-997f-22efc59f3dca
+status: experimental
+description: Detects potential JNDI Injection exploitation. Often coupled with Log4Shell exploitation.
+references:
+    - https://www.shiftleft.io/blog/log4shell-jndi-injection-via-attackable-log4j/
```

Comment:
```suggestion
```
get a 404

Reply:
find this https://secariolabs.com/research/analysing-and-reproducing-poc-for-log4j-2-15-0